packages/react-native-sdk/src/theme/theme.ts

```diff
@@ -585,4 +610,32 @@ export const defaultTheme: Theme = {
     buttonIcon: {},
     buttonText: {},
   },
+  getValue: function (component: string, prop: string) {
```

Comment:
@santhoshvai @oliverlaz What do you think about this option having a `getValue` method that will check if there is value for given prop for the given component and if there isn't to just fallback to some theme defaults value. It's usage would be: `backgroundColor: theme.getValue('button', 'backgroundColor'),`.

Reply:
It makes sense. I would even shorten the name of the method to `theme.get(path)`.

Also, we can support `theme.get('button.backgroundColor')`.

Reply:
The function is fine,

but its super confusing right now that I should pass `getValue("", "color")` to get the generic values.. 

also, I have a larger question, is this function necessary? Its already easier in my view

```
const theme = useTheme()
const defaultBg = theme.defaults. backgroundColor
const buttonBg = theme.button.backgroundColor
```

Reply:
@santhoshvai I see this function as alternative helper to your approach, so I think we can add it as additional way to read from theme and fallback to defaults.